### PR TITLE
[#56634] 14.4 Documentation: OpenProject Dark Mode  https://community…

### PR DESCRIPTION
**Pull request by MayaBerd**
Adding documentation on Dark mode

### CODE REVIEW

File: docs/getting-started/my-account/README.md [hunk 1]
```diff
@@ -76,6 +76,12 @@ This mode is recommended for users with visuals impairment.
 
 ![High contrast mode in OpenProject](openproject_my_account_high_contrast_mode.png)
 
+### Select the dark mode
+
+In the dropdown menu **Mode** you can pick the color mode. The default setting is the **Light mode**. You can also select the **Dark mode**, which will change the color theme of the OpenProject instance for you.
```
Comment (HDinger) [inline docs/getting-started/my-account/README.md]: I don't know whether this is the right place, but somewhere we should mention that the dark mode ignores parts of the configured design. Top header and sidemenu colors are completely ignored and overriden by us for this mode. Only the accent color and the primary button color are kept, but calculated to brighter variants.

Comment (MayaBerd) [reply docs/getting-started/my-account/README.md]: Thanks for the comment, I'll add it here. For now it's the only place where Dark mode is mentioned.

**Review by HDinger — APPROVED** [review]

File: docs/getting-started/my-account/README.md [hunk 2]
```diff
@@ -196,7 +206,9 @@ RSS tokens allow users to keep up with the latest changes in this OpenProject in
 
 Create a new token by clicking the **+RSS token** button. This will create your token and trigger a message showing you the access token.
 
-> **Note**: You will only be able to see the RSS access token once, directly after you create it. Make sure to copy it.
+> [!IMPORTANT]
+>
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
```

File: docs/getting-started/my-account/README.md [hunk 3]
```diff
@@ -209,7 +221,9 @@ To view and manage your OpenProject sessions navigate to **My account** and choo
 - **Current sessions**: here you can see all of your sessions. If for example you are logged into OpenProject from different browsers or devices, all will be shown in the list. Current session is the one you are currently using. You can terminate the sessions that are no longer in use. Inactive sessions will be removed from the list after 7 days (depending on the authentication settings they may become invalid earlier).
 - **Remembered devices**: here you can see a list of all devices that you are logged into using the "Stay logged in" option. You will have selected that option when [logging in](../sign-in-registration/). Whether or not that option is available and the duration of time for which you can stay logged in depends on the authentication settings of your instance.
 
-> **Note**: Closing a browser does not necessarily terminate the session. It might still be displayed in the list and will be reactivated if you open the browser. This depends on both your browser's and the OpenProject instance's settings.
+> [!NOTE]
+>
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
```

File: docs/getting-started/my-account/README.md [hunk 4]
```diff
@@ -233,7 +247,9 @@ Default: Enable daily email reminders: 2am, Monday - Friday.
 
 You can choose to receive emails immediately, or only on certain days and times, temporarily pause reminder emails, or opt for no reminders at all.
 
-> **Note**: if you have selected the *immediately when someone mentions me* option, you will only be notified once, i.e. this reminder will not be duplicated in a daily reminder. 
+> [!IMPORTANT]
+>
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
```

File: docs/getting-started/my-account/README.md [hunk 5]
```diff
@@ -233,7 +247,9 @@ Default: Enable daily email reminders: 2am, Monday - Friday.
...
+> if you have selected the *immediately when someone mentions me* option, you will only be notified once, i.e. this reminder will not be duplicated in a daily reminder.
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
> If you have selected the *immediately when someone mentions me* option, you will only be notified once, i.e. this reminder will not be duplicated in a daily reminder. 
```

File: docs/getting-started/my-account/README.md [hunk 6]
```diff
@@ -256,7 +272,9 @@ OpenProject uses Gravatar as default profile image. It displays a preview of you
 
 Also, you can upload a **Custom Avatar** by choosing a Avatar to be uploaded from a file. Press the blue **Update** button to change your profile picture.
 
-> **Note**: The optimum size to upload a new profile picture is 128 by 128 pixel. Larger files will be cropped.
+> [!TIP]
+>
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
```

File: docs/getting-started/my-account/README.md [hunk 7]
```diff
@@ -266,6 +284,8 @@ To delete your account, select **Delete account** from the side menu and enter y
 
 ![openproject_my_account_delete_account](openproject_my_account_delete_account.png)
 
-> **Important**: Deleting a user account is permanent and cannot be reversed.
+> [!WARNING]
+>
```
Comment (MayaBerd) [inline docs/getting-started/my-account/README.md]:
```suggestion
```

**Pull request merged**